perma_web/conftest.py

```diff
@@ -662,3 +676,58 @@ def one_two_three_dict():
     assert 'three' in data
     assert 'four' not in data
     return data
+
+
+#
+# Helpers
+#
+
+def randomize_capitalization(s):
+    return ''.join(choice((str.upper, str.lower))(c) for c in s)
+
+
+def log_in_user(client, user, password=TEST_USER_PASSWORD):
+    client.logout()
+    return client.post(reverse('user_management_limited_login'), {'username': user, 'password': password}, secure=True)
```

Comment:
Okay, I feel like I should be able to figure this out, but: do you know why we need both `log_in_user` and `attempt_login`? _Do_ we need both? If you aren't sure either... then I think we should try and see if we can get away with just `attempt_login`.

Reply:
Great! Will do.